_data/education.yml

```
@@ -247,6 +247,14 @@ websites:
       twitter: ISACANews
       facebook: ISACAHQ
 
+   - name: Karlsruher Institut für Technologie
```

```suggestion
    - name: Karlsruher Institut für Technologie
```
Travis can't find the start tag, when the yaml syntax isn't adhered to.

---

@PauleTV As you have not set your pull request to write access by members, I'd kindly ask you to commit my suggestion. Thanks!